src/test/scala/org/kibanaLoadTest/simulation/branch/TelemetryAPICachedJourney.scala

```diff
@@ -0,0 +1,39 @@
+package org.kibanaLoadTest.simulation.branch
+
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ScenarioBuilder
+import org.kibanaLoadTest.scenario.{Login, Home, TelemetryAPI}
+import org.kibanaLoadTest.simulation.BaseSimulation
+
+class TelemetryAPICachedJourney extends BaseSimulation {
+  val scenarioName = s"Branch cached telemetry journey ${appConfig.buildVersion}"
+
+  props.maxUsers = 400
```

Comment:
Probably 400 users are too much. We don't have that many users in a real-world scenario for a single Kibana instance, do we?

Reply:
At the moment we don't, 250-300 is more realistic boundary. 
But on the other hand, it is easier to spot performance degradation with higher load (based on passed experiments)

Reply:
For the cached scenario, IMO, the only critical situation is the first batch of 20 requests because that would likely have a fresh cache and it will need to generate the telemetry payload. After that, it's just a matter of responding from the cached value, so we can load it as much as we want (and the load test will actually test how much load our APIs can handle because the endpoint simply returns a value that it has in memory).

For the always-fresh scenario, I can see from the Jenkins build that the first set of timeouts occurs with 188 active users (still below our boundary):

```
================================================================================
2022-01-05 16:15:47                                         260s elapsed
---- Requests ------------------------------------------------------------------
> Global                                                   (OK=708    KO=30    )
> login                                                    (OK=444    KO=0     )
> telemetry: /api/telemetry/v2/clusters/_********             (OK=264    KO=30    )
---- Errors --------------------------------------------------------------------
> i.g.h.c.i.RequestTimeoutException: Request timeout to localhos     30 (100.0%)
t/127.0.0.1:5620 after 60000 ms

---- Branch telemetry journey 8.1.0-SNAPSHOT -----------------------------------
          active: 188    / done: 290   
================================================================================
```

To be fair, since the implementation of the cache, it's highly unlikely that we'll trigger those many fresh requests at once. However, I think that this load test can serve as a measure of how much pressure we put on ES & Kibana during the report generation. What do you think about setting `maxUsers` for the always-fresh scenario to 150? It should provide us with some buffer to accommodate some new _kind of_ expensive queries (in case they are implemented in the future), but it will raise the alerts if they are too much.

Reply:
I think that we could implement https://github.com/elastic/kibana/issues/122636 and improve this test to use that info to identify when collectors start to fail (different to failing from 60s timeouts). What do you think?